packages/bpk-component-calendar/src/bpk-calendar-date.scss

```diff
@@ -24,7 +24,7 @@
   padding: 0;
   border: none;
   border-radius: $bpk-border-radius-pill;
-  background-color: $bpk-calendar-day-background-color;
+  background-color: transparent;
```

Comment:
Should I remove this variable from bpk-tokens? It's not used anywhere.

Reply:
Yup its a good shout - what we have done previously is mark that token as deprecated and we'll remove it in a future release of `bpk-tokens`.

(FYI for other readers here is a link to the values of the tokens above: https://github.com/Skyscanner/backpack/blob/3df5ea4c52386a2e51acb428c0c716a965f9ed33/packages/bpk-tokens/src/web/base/calendar.json#L25)

Reply:
I added the deprecation markers.

Reply:
Unfortunately you didn't run `npm run build` before committing :P

Reply:
But I did realize it before you wrote that comment. So partial credit? :)

Reply:
Absolutely 😄